From 5f78bb7df0fbee84ab250c87cb079765aefbd4a1 Mon Sep 17 00:00:00 2001
From: Dinolek <git@dinolek.me>
Date: Wed, 9 Jul 2025 15:58:28 +0200
Subject: [PATCH 065/115] dt-bindings: display: mediatek: Add support for
 MT6785

Add MT6785 compatibles to AAL, CCORR, COLOR, DITHER, DSI, GAMMA, OVL,
POSTMASK and RDMA.

Signed-off-by: Dinolek <git@dinolek.me>
---
 .../devicetree/bindings/display/mediatek/mediatek,aal.yaml       | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,ccorr.yaml     | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,color.yaml     | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,dither.yaml    | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,dsi.yaml       | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,gamma.yaml     | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,ovl-2l.yaml    | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,ovl.yaml       | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,postmask.yaml  | 1 +
 .../devicetree/bindings/display/mediatek/mediatek,rdma.yaml      | 1 +
 10 files changed, 10 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,aal.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,aal.yaml
index 4bbea72b2..debc10c11 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,aal.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,aal.yaml
@@ -37,6 +37,7 @@ properties:
           - const: mediatek,mt8173-disp-aal
       - items:
           - enum:
+              - mediatek,mt6785-disp-aal
               - mediatek,mt8186-disp-aal
               - mediatek,mt8188-disp-aal
               - mediatek,mt8192-disp-aal
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,ccorr.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,ccorr.yaml
index 5c5068128..fab350b03 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,ccorr.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,ccorr.yaml
@@ -31,6 +31,7 @@ properties:
           - const: mediatek,mt8183-disp-ccorr
       - items:
           - enum:
+              - mediatek,mt6785-disp-ccorr
               - mediatek,mt8186-disp-ccorr
               - mediatek,mt8188-disp-ccorr
               - mediatek,mt8195-disp-ccorr
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,color.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,color.yaml
index 5564f4063..864caad2f 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,color.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,color.yaml
@@ -38,6 +38,7 @@ properties:
           - const: mediatek,mt2701-disp-color
       - items:
           - enum:
+              - mediatek,mt6785-disp-color
               - mediatek,mt6795-disp-color
               - mediatek,mt8183-disp-color
               - mediatek,mt8186-disp-color
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,dither.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,dither.yaml
index 891c95be1..ba5486730 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,dither.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,dither.yaml
@@ -26,6 +26,7 @@ properties:
           - mediatek,mt8183-disp-dither
       - items:
           - enum:
+              - mediatek,mt6785-disp-dither
               - mediatek,mt8167-disp-dither
               - mediatek,mt8186-disp-dither
               - mediatek,mt8188-disp-dither
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,dsi.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,dsi.yaml
index 27ffbccc2..801e7a19a 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,dsi.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,dsi.yaml
@@ -36,6 +36,7 @@ properties:
           - const: mediatek,mt8173-dsi
       - items:
           - enum:
+              - mediatek,mt6785-dsi
               - mediatek,mt8195-dsi
               - mediatek,mt8365-dsi
           - const: mediatek,mt8183-dsi
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,gamma.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,gamma.yaml
index ec1054bb0..645583158 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,gamma.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,gamma.yaml
@@ -32,6 +32,7 @@ properties:
           - const: mediatek,mt8173-disp-gamma
       - items:
           - enum:
+              - mediatek,mt6785-disp-gamma
               - mediatek,mt8186-disp-gamma
               - mediatek,mt8188-disp-gamma
               - mediatek,mt8192-disp-gamma
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl-2l.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl-2l.yaml
index ac0d924a4..8de3275af 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl-2l.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl-2l.yaml
@@ -26,6 +26,7 @@ properties:
           - mediatek,mt8192-disp-ovl-2l
       - items:
           - enum:
+              - mediatek,mt6785-disp-ovl-2l
               - mediatek,mt8186-disp-ovl-2l
           - const: mediatek,mt8192-disp-ovl-2l
 
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl.yaml
index 4df5c7b41..8f229436f 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,ovl.yaml
@@ -40,6 +40,7 @@ properties:
           - const: mediatek,mt8173-disp-ovl
       - items:
           - enum:
+              - mediatek,mt6785-disp-ovl
               - mediatek,mt8186-disp-ovl
               - mediatek,mt8365-disp-ovl
           - const: mediatek,mt8192-disp-ovl
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,postmask.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,postmask.yaml
index fb6fe4742..aa5118552 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,postmask.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,postmask.yaml
@@ -25,6 +25,7 @@ properties:
           - mediatek,mt8192-disp-postmask
       - items:
           - enum:
+              - mediatek,mt6785-disp-postmask
               - mediatek,mt8186-disp-postmask
               - mediatek,mt8188-disp-postmask
           - const: mediatek,mt8192-disp-postmask
diff --git a/Documentation/devicetree/bindings/display/mediatek/mediatek,rdma.yaml b/Documentation/devicetree/bindings/display/mediatek/mediatek,rdma.yaml
index d914c0664..741487f17 100644
--- a/Documentation/devicetree/bindings/display/mediatek/mediatek,rdma.yaml
+++ b/Documentation/devicetree/bindings/display/mediatek/mediatek,rdma.yaml
@@ -44,6 +44,7 @@ properties:
           - const: mediatek,mt8173-disp-rdma
       - items:
           - enum:
+              - mediatek,mt6785-disp-rdma
               - mediatek,mt8186-disp-rdma
               - mediatek,mt8192-disp-rdma
               - mediatek,mt8365-disp-rdma
-- 
2.56.0

